td-logger/src/logger.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+
+//! A standalone logger to directly log messages, independent of the log crate.
+
+use core::fmt;
+use lazy_static::lazy_static;
+use spin::Mutex;
+
+use crate::{dbg_port_write, dbg_write_string};
+
+pub const LOG_LEVEL_VERBOSE: usize = 1000;
+pub const LOG_LEVEL_INFO: usize = 100;
+pub const LOG_LEVEL_WARN: usize = 10;
+pub const LOG_LEVEL_ERROR: usize = 1;
+pub const LOG_LEVEL_NONE: usize = 0;
+
+pub const LOG_MASK_COMMON: u64 = 0x1;
```

Comment:
What does this actually log?

Reply:
Something "generic", or "other", maybe:)
Depend on the caller.